src/app_core.py

```diff
@@ -123,6 +123,13 @@ def strdef(name, default):
 app.config["CAPTCHA_SECRETKEY"] = os.getenv("CAPTCHA_SECRETKEY", None)
 app.config["CAPTCHA_SITEKEY"] = os.getenv("CAPTCHA_SITEKEY", None)
 
+app.config["THRESHOLD_TOTAL_SPENDABLE"] = os.getenv("THRESHOLD_TOTAL_SPENDABLE")
```

Comment:
these should be parsed as ints/floats here

Reply:
changed to what was suggested above.